### PR TITLE
Merge with up to date SwarrotBundle

### PR DESCRIPTION
**Pull request by conradfr**
Sorry for the squash commit.

FWIW I ran https://github.com/wimg/PHPCompatibility with "--runtime-set testVersion 5.3" to help ensure it works on PHP 5.3

### CODE REVIEW

File: Broker/AmqpLibFactory.php [hunk 1]
```diff
@@ -87,15 +87,15 @@ protected function getChannel($connection)
...
-               foreach ($this->connections[$connection]['ssl_options'] as $key => $value) {
-                   if (!empty($value)) {
+                $ssl_opts = array();
+                foreach ($this->connections[$connection]['ssl_options'] as $key => $value) {
+                    if (!empty($value)) {
                         $ssl_opts[$key] = $value;
-                   }
-               }
+                    }
+                }
```
Comment (FlorianBaba) [inline Broker/AmqpLibFactory.php]: He he, indent addict :+1:

File: Broker/BlackholePublisher.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Swarrot\SwarrotBundle\Broker;
+
+use Swarrot\Broker\Message;
+
+class BlackholePublisher extends Publisher
```
Comment (mlages) [inline Broker/BlackholePublisher.php]: holy sh*t, it's a trap !

**Comment by olaurendeau** [comment]
:+1:

**Comment by FlorianBaba** [comment]
:+1:

**Pull request merged**